lib/vagrant-libvirt/action.rb

```diff
@@ -54,7 +54,9 @@ def self.action_up
                 # b2.use SyncFolders
               end
             else
-              b2.use action_start
+              start_env = env.dup
+              start_env[:halt_on_error] = true
+              env[:action_runner].run(Action.action_start, start_env)
```

Comment:
I don't think you need to duplicate the env and use the action runner here, could be wrong though. Believe that is only needed for the halt/destroy on error actions because the env needs to be isolated for the destroy action to remove the interrupted key so it doesn't cause the middleware to abort the destroy/halt action.

Don't need the same for the start action, just set the env key and call as before.